src/Package/Impl/Help/ShowHelpOnCurrentCommand.cs

```diff
@@ -32,8 +34,10 @@ internal sealed class ShowHelpOnCurrentCommand : PackageCommand {
         private readonly IRInteractiveWorkflow _workflow;
         private readonly IActiveWpfTextViewTracker _textViewTracker;
 
+        private IActiveRInteractiveWindowTracker _activeReplTracker;
         public ShowHelpOnCurrentCommand(IRInteractiveWorkflow workflow, IActiveWpfTextViewTracker textViewTracker) :
             base(RGuidList.RCmdSetGuid, RPackageCommandId.icmdHelpOnCurrent) {
+            _activeReplTracker = VsAppShell.Current.ExportProvider.GetExportedValue<IActiveRInteractiveWindowTracker>();
```

Comment:
please provide it as ctor parameter so command can be tested without stub injection into VsTestCompositionCatalog